content/20.treatments.md

```diff
@@ -337,8 +332,11 @@ https://www.ncbi.nlm.nih.gov/pubmed/28099856
 https://www.ncbi.nlm.nih.gov/pubmed/27911847
 -->
 The lack of benefits indicated by large studies of COVID-19 treatment with HCQ (e.g., [@doi:10/ggwzsb]) is important to recognize clinically because these medications are also associated with dangerous side effects like prolonged QT interval [@doi:10.1155/2016/4626279].
+
 There is now evidence of increased mortality among older men receiving HCQ, and administration of HCQ and HCQ+AZ did not decrease the use of mechanical ventilation in the admitted patients [@doi:10.1101/2020.04.16.20065920]. 
-Even otherwise healthy patients had an increased risk of _de novo_ ventricular arrhythmia [@doi:10/ggwzsb].
+Although [@doi:10/ggwzsb] was initially thought to show that healthy patients had an increased risk of _de novo_ ventricular arrhythmia, this study has now been retracted. 
+An important concern not disclosed in the original publication was that whether the patients administered with HCQ/CQ had similar viral load or similar symptoms as the controls.
+There have still not been any studies that investigated HCQ/CQ efficacy using randomized controls.
```

Comment:
I'm not following all the HCQ literature, but is this new sentence still accurate now that https://www.nejm.org/doi/full/10.1056/NEJMoa2016638 is out?

> We conducted a randomized, double-blind, placebo-controlled trial across the United States and parts of Canada testing hydroxychloroquine as postexposure prophylaxis. We enrolled adults who had household or occupational exposure to someone with confirmed Covid-19 at a distance of less than 6 ft for more than 10 minutes while wearing neither a face mask nor an eye shield (high-risk exposure) or while wearing a face mask but no eye shield (moderate-risk exposure). Within 4 days after exposure, we randomly assigned participants to receive either placebo or hydroxychloroquine (800 mg once, followed by 600 mg in 6 to 8 hours, then 600 mg daily for 4 additional days). The primary outcome was the incidence of either laboratory-confirmed Covid-19 or illness compatible with Covid-19 within 14 days.

Reply:
@agitter I was actually just about to submit a new PR to address this and add the info from #366 !